libs/blocks/merch/upgrade.js

```diff
@@ -14,15 +14,23 @@ const isProductFamily = (offer, pfs) => {
   const productFamily = offer?.offer?.product_arrangement?.family;
   return productFamily && pfs.includes(productFamily);
 };
+export const LANA_OPTIONS = {
+  clientId: 'merch-at-scale',
+  sampleRate: 100,
```

Comment:
I know that normally the good practise is to set 1% not 100, but in this case I though we are not risking too many messages since 'Upgrade Now' is unlikely clicked by many users.
Also if we set it to less than 100% we will not see step by step interactions of user since there is a chance that not all events per user will be logged:
opening modal, redirect to the link, closing modal etc.

however i am fine to set it for lower percentage for now since we don't have issues reported.
we can change a value back if we will need to investigate something.
@yesil wdyt?
1%?